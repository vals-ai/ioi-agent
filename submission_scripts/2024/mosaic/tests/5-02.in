version https://git-lfs.github.com/spec/v1
oid sha256:2da5917b740e382d26e86c0722d6ccbea98bef150fd23e80760cdba8e1543722
size 5954539
