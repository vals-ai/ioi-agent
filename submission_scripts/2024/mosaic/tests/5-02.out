version https://git-lfs.github.com/spec/v1
oid sha256:f9128edae0f7123642a4de239506cfd950a51280d5f4555a9e519749b8f9a872
size 2138288
